spec/results/proto-google.yml

```diff
@@ -346,60 +377,60 @@ networks:
     static:
     - 10.99.0.25
   type: manual
+- name: test-network
+  subnets:
+  - cloud_properties:
+      ephemeral_external_ip: false
```

Comment:
`- name: default` and `-name: test-network` should be merged.

Reply:
@dennisjbell could you look at this spruce merge issue

Reply:
not a spruce merge issue
was due to line
```
networks:
- name: (( grab params.google_network_name)
```
which should be named default